spec/services/hyrax/custom_queries/find_file_metadata_spec.rb

```diff
@@ -0,0 +1,11 @@
+# rubocop:disable RSpec/EmptyExampleGroup
+RSpec.describe Hyrax::CustomQueries::FindFileMetadata do
+  let(:query_service) { Hyrax.query_service }
+  let(:valk_id) { double } # TODO: Stubbed waiting for tests
+  subject { query_service.custom_queries.find_file_metadata_by(id: valk_id) }
+
+  describe '.find_file_metadata_by' do
+    # TODO: Need tests for this custom query once inMemory adapter is supported
```

Comment:
if we're not going to write the tests, i think i'd prefer to either drop the file altogether, or write a few pending tests we intend to fill in later:

```ruby
describe '.find_file_metadata_by' do
  it 'does thing we know we want to test'
  it 'does other thing'
end
```

this way it's clear what we *should* be testing here, and what tech debt we have taken on.

Reply:
that said, i think it would be really nice to just slot in tests using the memory adapter and its query service.

Reply:
@no-reply I went ahead and wrote the in-memory tests for this custom query